inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.h

```diff
@@ -14,6 +14,7 @@
 namespace MKLDNNPlugin {
 
 #define MAX_ELTWISE_INPUTS 7
+#define MAX_DIM_RANK 12
```

Comment:
In general I am ok with this solution for now. However we need to check that execute will not receive bigger rank. Please extend isSupportedOperation with corresponding check. We also need to make sure that during mkldnn graph optimization Eltwise with bigger rank won't be fused.

Reply:
ReferenceExecution isn't depended on these indexes. So I added condition for variable `isOptimized` but not in isSupportedOperation and I added conditions for fusing optimizations. Thanks!